doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3435,6 +3435,13 @@ The general command to add a hint to some databases :n:`{+ @ident}` is
 
       Declares each :n:`@ident` as a transparent or opaque constant.
 
+   .. cmdv:: Hint %( Variables %| Constants %) %( Transparent %| Opaque %)}
+      :name: Hint ( Variables | Constants )
+
+      This sets the transparency flag used during unification of
+      hints in the database for all constants or all variables,
+      overwritting the existing settings of opacity.
```

Comment:
Formulated like this it sounds like it overwrites the previous `Hint Transparent` / `Hint Opaque` commands. In the case of constants, I suppose you rather mean: this sets the default transparency for constants without a `Hint Transparent` / `Hint Opaque` specific definition.

Reply:
So after having looked at the code, I see that this does overwrite the existing settings of opacity (the previous `Hint Transparent` / `Hint Opaque` commands. So I suggest you be even more explicit by saying this and by strongly encouraging people to use `Hint Variables` / `Hint Constants` right after the `Create HintDb` command.